frontend/model/contracts/chatroom.js

```diff
@@ -107,7 +107,7 @@ function messageReceivePostEffect ({
 
   shouldNotifyMessage && makeNotification({
     title,
-    body: swapUserIDForUsername(text),
+    body: messageType === MESSAGE_TYPES.TEXT ? swapUserIDForUsername(text) : L('New message'),
```

Comment:
@taoeffect 
This is the fix for the issue #1953 .
The cause of the issue was that the `text` here is only a valid value when the message is of type `MESSAGE_TYPES.TEXT`. for the rest of the types (e.g. `MESSAGE_TYPES.POLL`), the browser notification message will say `New message` as the default. But let me know if you have a better idea though.